plugins/events/init.go

```diff
@@ -26,6 +26,13 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Version:     1,
 			RequestType: utils.GetTypeString((*events.Webhook)(nil)),
 			HandlerFunc: webhookAddHandler},
+		route.Route{
+			Name:        "WebhookTest",
+			Method:      "POST",
+			Pattern:     "/events/webhook/test",
+			Version:     1,
+			RequestType: utils.GetTypeString((*events.Webhook)(nil)),
+			HandlerFunc: webhookTestHandler},
```

Comment:
Hmm. WebhookTest sounds like this is for test code but it looks like its a permanent part of the server. What's this actually do?

Reply:
This API is to test that if the webhook can be reached from all gluster nodes or not